node/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
```

Comment:
Thoughts on having `foundryup` add a binary to `~/.foundry/bin/` called `anvil` that serves as an alias for `forge node`? This feels like a third tool (in addition to `forge` and `cast`) so I expected to be able to start this by running `anvil` as opposed to `forge node`

Reply:
I think that's planned, 
starting it is very simple, see `node.rs` command

Reply:
yeah let's do as `anvil` without `forge node` tbh, keeps the `forge` binary smaller as well